joomla/joomla-cms
# Flatten UI

Pull Request for Issue # .

### Summary of Changes
Flatten UI styling in the Protostar frontend template to match backend.

If this is an improvement or not is questionable. I am happy enough closing if deemed not.

### Testing Instructions
Apply PR and navigate frontend. In particular check frontend article and module editing.


### Before
![image](https://user-images.githubusercontent.com/2803503/46150298-23539b00-c264-11e8-817c-2f67dbb961bc.png)



### After
![image](https://user-images.githubusercontent.com/2803503/46150255-1040cb00-c264-11e8-8f95-7eabda1f58f5.png)
